generators/kubernetes-helm/templates/app/chart.yml.ejs

```diff
@@ -0,0 +1,25 @@
+name: <%= app.baseName.toLowerCase() %>
+version: <%= jhipsterVersion %>
```

Comment:
I don't understand why it's the jhipster version here and not the app's version.

Reply:
We can put app version as well. Thought as we are generating the helm package, a specific version of our release might fit well here.